ci/generate_tpch_data.py

```
@@ -0,0 +1,46 @@
+    )
+    parser.add_argument(
+        "--output-dir",
+        default=os.environ.get("TPCH_DATA_DIR"),
+        help="Output directory. Defaults to TPCH_DATA_DIR environment variable.",
+    )
+    args = parser.parse_args()
+
+    if args.output_dir is None:
+        parser.error("--output-dir is required (or set TPCH_DATA_DIR).")
+
+    subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)